### ISSUE TITLE
Enhancement: option to start keyboard selection at X

### ISSUE THREAD

**Issue by phedders** [johanmalm/jgmenu#130]
It would be nice if there was an option such as --highlight=X where entry X would start highlighted so that pressing enter would select that option and cursor keys would move from that position.

Many thanks

**Comment by johanmalm** [comment]
How do you suggest we implement?
`--highlight=3` to select 3rd item?
`--highlight=foo` to select first item which contains foo in "Name" (I think this would probably make most sense)
or something else.

**Comment by dumblob** [comment]
Something like https://github.com/joewing/jwm/issues/102#issuecomment-126894504 - technically e.g. as proposed in https://github.com/joewing/jwm/issues/252#issuecomment-263536860 .

----

Btw. does jgmenu support the "triangle timeout" as linked from https://github.com/joewing/jwm/issues/522 ?

**Comment by johanmalm** [comment]
Yes, "triangle timeout" is supported (although I've never heard it called that before). See hover_delay in jgmenu(1).

Regarding starting selection at X, I'm happy to merge well-designed + tested patches for this. It should be relatively straight forward to implement. If it were me, I'd start with something similar to the `--checkout` option and just set `menu.sel` to it.
